extensions/vscode/src/granite/ollama/ollamaServer.ts

```diff
@@ -140,6 +177,7 @@ export class OllamaServer implements IModelServer, Disposable {
         installCommand = [
           'clear',
           'set -e',  // Exit immediately if a command exits with a non-zero status
+          'killall "Ollama"',
```

Comment:
there's a bug here, if the ollama process is already stopped, the command will fail